is_core/auth_token/models.py

```diff
@@ -41,8 +41,14 @@ class Token(models.Model):
     expiration = models.BooleanField(null=False, default=True)
     ip = models.GenericIPAddressField(null=False, blank=False)
     auth_slug = models.SlugField(null=True, blank=True)
+    backend = models.CharField(max_length=255, null=False, blank=False)
+
     is_from_header = False
 
+    @property
+    def active_takeover(self):
+        return self.user_takeovers.filter(is_active=True).last()
```

Comment:
Trochu blbé je, že jich může být víc a tahle property budí dojem, že může být aktivní pouze jeden takeover. Nemůže se stát, že když budou mít ten token převzatí 2 uživatelé a jeden z nich se odhlásí, odhlásí to místo něj toho druhého?